### PR TITLE
docs: add note about migration in #11098 to Changelog

### PR DESCRIPTION
**Pull request by betodealmeida**
### SUMMARY

Add entry to `UPDATING.md` about the DB migration in https://github.com/apache/incubator-superset/pull/11098.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

N/A

### TEST PLAN

N/A

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by mistercrunch — APPROVED** [review]

File: UPDATING.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ assists people when migrating to a new version.
 
 * [11244](https://github.com/apache/incubator-superset/pull/11244): The `REDUCE_DASHBOARD_BOOTSTRAP_PAYLOAD` feature flag has been removed after being set to True for multiple months.
 
+* [11098](https://github.com/apache/incubator-superset/pull/11098): includes a database migration that adds a column caled `uuid` to all models, and updates the `Dashboard.position_json` column to also have UUIDs of charts. Depending on the number of objects the migration might take up to 3-4 minutes, requiring planning for the downtime.
```
Comment (mistercrunch) [inline UPDATING.md:28]: NIT: "to all models" -> "to some models" (?)

Comment (ktmud) [reply UPDATING.md:28]: nit: I'm no native speaker, but the two "the"'s in the last sentence seems not necessary?

Comment (ktmud) [inline UPDATING.md:28]:
```suggestion
* [11098](https://github.com/apache/incubator-superset/pull/11098): includes a database migration that adds a `uuid` column to most models, and updates `Dashboard.position_json` to include chart UUIDs. Depending on number of objects, the migration may take up to 5 minutes, requiring planning for downtime.
```

**Review by ktmud — COMMENTED** [review]
Sorry for nitpicking!

**Comment by betodealmeida** [comment]
> Sorry for nitpicking!

No need to apologize, it's much better. I also noticed I had a typo in "called", I wrote "caled". :-P

**Review by ktmud — APPROVED** [review]

**Pull request merged**